diff --git a/es/components/Card/style.js b/es/components/Card/style.js
index 3751792bef1c5d24795c738006ae2ff38595ceb0..adb82dfa23b577aed155543255ff01d080c1bd64 100644
--- a/es/components/Card/style.js
+++ b/es/components/Card/style.js
@@ -24,8 +24,7 @@ var genProCardStyle = function genProCardStyle(token) {
     borderRadius: token.borderRadius
   }, resetComponent === null || resetComponent === void 0 ? void 0 : resetComponent(token)), {}, (_objectSpread2 = {
     '*': {
-      boxSizing: 'border-box',
-      fontFamily: token.fontFamily
+      boxSizing: 'border-box'
     },
     '&-box-shadow': {
       boxShadow: '0 1px 2px -2px #00000029, 0 3px 6px #0000001f, 0 5px 12px 4px #00000017',
diff --git a/lib/components/Card/style.js b/lib/components/Card/style.js
index 8d25066eac53170cfdb4bbb1c43632455ae8e9b2..da57aefb4bbbb725222c157dd73f755bb774a505 100644
--- a/lib/components/Card/style.js
+++ b/lib/components/Card/style.js
@@ -31,8 +31,7 @@ var genProCardStyle = function genProCardStyle(token) {
     borderRadius: token.borderRadius
   }, _proUtils.resetComponent === null || _proUtils.resetComponent === void 0 ? void 0 : (0, _proUtils.resetComponent)(token)), {}, (_objectSpread2 = {
     '*': {
-      boxSizing: 'border-box',
-      fontFamily: token.fontFamily
+      boxSizing: 'border-box'
     },
     '&-box-shadow': {
       boxShadow: '0 1px 2px -2px #00000029, 0 3px 6px #0000001f, 0 5px 12px 4px #00000017',
diff --git a/lib/components/StatisticCard/index.js b/lib/components/StatisticCard/index.js
index 18752b6088becea591687c5698d72becffb76b6c..60843642aa9bca953c1c1f603ad88a127525161e 100644
--- a/lib/components/StatisticCard/index.js
+++ b/lib/components/StatisticCard/index.js
@@ -17,8 +17,8 @@ var _Divider = _interopRequireDefault(require("../Divider"));
 var _Operation = _interopRequireDefault(require("../Operation"));
 var _Statistic = _interopRequireDefault(require("../Statistic"));
 var _style = require("./style");
-require("antd/es/divider/style");
-require("antd/es/statistic/style");
+require("antd/lib/divider/style");
+require("antd/lib/statistic/style");
 var _excluded = ["children", "statistic", "className", "chart", "chartPlacement", "footer"];
 var StatisticCard = function StatisticCard(props) {
   var _classNames;
diff --git a/lib/index.js b/lib/index.js
index bbc74612fb3c4487fd73d9b7250993473c1e4dd6..81e48af1a9fc305d77df3512b51cbaec45b42e6c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -33,6 +33,6 @@ var _CheckCard = _interopRequireDefault(require("./components/CheckCard"));
 var _Statistic = _interopRequireDefault(require("./components/Statistic"));
 var _StatisticCard = _interopRequireDefault(require("./components/StatisticCard"));
 var _ProCard = _interopRequireDefault(require("./ProCard"));
-require("antd/es/card/style");
+require("antd/lib/card/style");
 var _default = _ProCard.default;
 exports.default = _default;
\ No newline at end of file
